internal/transactionsubmission/wallet_creation_transaction_handler.go

```diff
@@ -194,17 +192,6 @@ func (h *WalletCreationTransactionHandler) BuildInnerTransaction(ctx context.Con
 		return nil, applyErr
 	}
 
-	txParams.BaseFee = h.calculateAdjustedBaseFee(simulationResponse)
-
-	channelAccount, err := h.engine.HorizonClient.AccountDetail(horizonclient.AccountRequest{AccountID: txJob.ChannelAccount.PublicKey})
-	if err != nil {
-		return nil, utils.NewHorizonErrorWrapper(err)
-	}
-	txParams.SourceAccount = &txnbuild.SimpleAccount{
-		AccountID: txJob.ChannelAccount.PublicKey,
-		Sequence:  channelAccount.Sequence,
-	}
-
```

Comment:
Removed this part because we already had the sequence setup a few lines above, so this is not needed.